### PR TITLE
MKM: Judith, Carnage Connoisseur + Imp token

### PR DESCRIPTION
**Pull request by MikeS-NZ**
MKM: Add card script for new MKM card Judith, Carnage Connoisseur, plus her Imp token.

- Judith, Carnage Connoisseur
- Red 2/2 Imp creature token with "When this creature dies, it deals 2 damage to each opponent."

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/judith_carnage_connoisseur.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name:Judith, Carnage Connoisseur
+ManaCost:3 B R
+Types:Legendary Creature Human Shaman
+PT:3/4
+T:Mode$ SpellCast | TriggerZones$ Battlefield | ValidCard$ Instant,Sorcery | ValidActivatingPlayer$ You | NoResolvingCheck$ True | Execute$ TrigCharm | TriggerDescription$ Whenever you cast an instant or sorcery spell, ABILITY
+SVar:TrigCharm:DB$ Charm | Choices$ DBPump,DBToken
+SVar:DBPump:DB$ Pump | Defined$ TriggeredCard | KW$ Deathtouch & Lifelink | PumpZone$ Stack | SpellDescription$ That spell gains deathtouch and lifelink.
+SVar:DBToken:DB$ Token | TokenAmount$ 1 | TokenScript$ r_2_2_imp_burn_opp | TokenOwner$ You | SpellDescription$ Create a 2/2 red Imp creature token with "When this creature does, it deals 2 damage to each opponent."
```
Comment (squee1968) [inline forge-gui/res/cardsfolder/upcoming/judith_carnage_connoisseur.txt:8]: Typo - does should be dies

File: forge-gui/res/cardsfolder/upcoming/judith_carnage_connoisseur.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+DeckNeeds:Type$Instant|Sorcery
+SVar:BuffedBy:Spell.Instant,Spell.Sorcery
+Oracle:Whenever you cast an instant or sorcery spell, choose one —\n• That spell gains deathtouch and lifelink.\n• Create a 2/2 red Imp creature token with "When this creature does, it deals 2 damage to each opponent."
```
Comment (squee1968) [inline forge-gui/res/cardsfolder/upcoming/judith_carnage_connoisseur.txt:11]: Typo - does should be dies

Comment (MikeS-NZ) [reply forge-gui/res/cardsfolder/upcoming/judith_carnage_connoisseur.txt]: Fixed in both places! Thanks

File: forge-gui/res/tokenscripts/r_2_2_imp_burn_opp.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Name:Imp Token
+ManaCost:no cost
+Types:Creature Imp
+Colors:red
+PT:2/2
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ ImpTrigDamage | TriggerDescription$ When this creature dies, it deals 2 damage to each opponent.
+SVar:ImpTrigDamage:DB$ DealDamage | Defined$ Player.Opponent | NumDmg$ 2
```
Comment (Northmoc) [inline forge-gui/res/tokenscripts/r_2_2_imp_burn_opp.txt:7]: `Defined$ Opponent` is a bit better

Comment (MikeS-NZ) [reply forge-gui/res/tokenscripts/r_2_2_imp_burn_opp.txt]: Fixed!

**Review by Northmoc — APPROVED** [review]
Looks good!

**Pull request merged**